README.md

```
@@ -51,6 +50,12 @@ When `PORT` or `QUERYPORT` are altered this change MUST reflect the port mapping
 
 The port for players to connect is now **10002**.
 
+### How to customize SCUM server
+
+All server data is exposed in the `scumserver-data` folder.  
+For further information check [here](https://www.google.com/search?q=scum+server+settings).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)